### PR TITLE
new icon: dreamweaver (original, plain, line)

### PR DESCRIPTION
**Pull request by PantheraRed**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**
#1711 

## Link to prove your SVG is correct and up-to-date.
https://en.wikipedia.org/wiki/Adobe_Dreamweaver#/media/File:Adobe_Dreamweaver_CC_icon.svg

### CODE REVIEW

**Comment by PantheraRed** [comment]
Checked everywhere. Not sure what am I missing.

File: devicon.json [hunk 1]
```diff
@@ -2393,6 +2393,29 @@
         "color": "#623697",
         "aliases": []
     },
+    {
+        "name": "dreamweaver",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:2398]:
```suggestion
        "altnames": [
            "Adobe Dreamweaver"
        ],
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Looks good to me. 
You can ignore the failed optimizer check, it's failing due to permission issues.

Only suggestion is adding the full name as an altname

**Review by Snailedlt — APPROVED** [review]
Fastest developer in the west 🤠

LGTM! ✔️ 
Thank you for another great contribution! 🙇

**Pull request merged**